README.md

```diff
@@ -87,6 +88,21 @@ rake argo:repo:load
 rspec
 ```
 
+The continuous integration build can also be run by:
```

Comment:
would be nice to add a warning explaining that running CI (or anything that runs `argo:jetty:clean`) will reinstall your jetty instance and blow away any test data you might have setup.